### PR TITLE
css layout classes - Chris Tran

### PR DESCRIPTION
**Pull request by tran-christopher**
Animation

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.container {
+  display: flex;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
```
Comment (shawnkost) [inline css-layout-classes/styles.css]: Your `.container` shouldn't have any properties related to `flex` on here. This won't necessarily always be the case, but just for this specific exercise.

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
```
Comment (shawnkost) [inline css-layout-classes/styles.css]: Delete this, we aren't doing anything with `justify-content` here

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Chris!

**Comment by tran-christopher** [comment]
Animation

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**